cloud-watch-to-slack-testing/deployment/index.js

```diff
@@ -291,12 +291,20 @@ function setSlackChannelBasedOnSNSTopic(topicArn) {
 
 function processEvent(event, callback) {
   console.log(event);
-  const message = JSON.parse(event.Records[0].Sns.Message);
+  let message;
+  let messageText;
+  try {
+    message = JSON.parse(event.Records[0].Sns.Message);
+    messageText = messageTextFromMessageObject(message);
+  } catch (e) {
+    message = event.Records[0].Sns.Message;
+    console.log("Plain message received: " + messageText);
```

Comment:
@david4096 should this have been `+ message`? as the `messageText` var was not set yet?

Reply:
dangit

Reply:
😇